tools/bbbsupervisor/bbbsupervisor.go

```diff
@@ -45,23 +45,44 @@
 package main
 
 import (
+	"encoding/json"
 	"flag"
 	"fmt"
+	"net/http"
 	"os"
 	"os/exec"
 	"strings"
+	"time"
 )
 
 type (
```

Comment:
as a matter of personal taste only, I like type definitions without bundling in `type()` to save indentation.

Reply:
I have the opposite taste (bundling `type`, `var`, `const` etc so it's all neatly together).. so I guess until we have a style guide to refer to, we'll need to flip a coin or something. :)

I'll leave it as-is for now.